scripts/roles/todd/vars/toddvars.yml

```diff
@@ -0,0 +1,10 @@
+docker_repo: 'deb https://apt.dockerproject.org/repo {{ ansible_distribution|lower }}-{{ ansible_distribution_release|lower }} main'

```

Comment:
Yeah, for now I'm only using this role for organization purposes for standing up the Vagrant machine, so I think the current location is fine. I added the hyphens in 0db2172; good catch